WhaleSpotting/react-app/src/styles/PageNav.scss

```diff
@@ -0,0 +1,31 @@
+@import "./constants.scss";
+
+.page-nav {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    font-family: $fontFamily;
+
+    .page-button{
+        height: 50px;
+        width: 25%;
+        font-family: $fontFamily;
+        background-color: $primaryLight;
+        border-radius: 10px;
+        border-width: 0px;
+        color: $textOnPrimary;
+
+        &:hover 
+        {
+            background-color: $primaryCard;
```

Comment:
Like the other button, hover should maybe go to primaryColour

Reply:
fixed